docs/quickstarts/excel-custom-functions-quickstart.md

```diff
@@ -0,0 +1,109 @@
+---
+ms.date: 03/06/2019
+description: Developing custom functions in Excel quick start guide.
+title: Custom functions quick start (preview)
+localization_priority: Normal
+---
+
+# Get started developing Excel Custom Functions
+
+Custom functions enable developers to add new functions to Excel by defining those functions in JavaScript or Typescript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel, such as `SUM()`.
```

Comment:
```suggestion
With custom functions, developers can now add new functions to Excel by defining them in JavaScript or Typescript as part of an add-in. Excel users can access custom functions just as they would any native function in Excel, such as `SUM()`.
```

Reply:
Per Microsoft Style Writing Guide, "Don't use enables (or allows or lets) to refer to things that Microsoft or Microsoft apps make possible for the customer"